test/index.test.js

```diff
@@ -16,55 +16,51 @@ const CLIENT_NAME = 'client_name'
 const ANOTHER_DATABASE_NAME = 'my_awesome_database'
 const COLLECTION_NAME = 'mycoll'
 
-t.Test.prototype.addAssert('objectId', 1, function (ObjectId, message, extra) {
+function objectIdTest (t, ObjectId, message) {
```

Comment:
If you plan to nest all this helper function inside `t.test`, please also specify the `t.plan` inside.

Reply:
added `t.plan` in all 3 functions